docs/sdk/cloud_storage.md

```diff
@@ -1,13 +1,17 @@
 # Cloud storage module
 
-## Data Integration versus Data Connection
+!!! warning "Beta feature"
+    The cloud storage feature is currently in beta. If you encounter any issue, do not hesitate to contact [Kili support](mailto:support@kili-technology.com) or create an issue on [GitHub](https://github.com/kili-technology/kili-python-sdk/issues).
```

Comment:
The cloud storage feature is currently in alpha. It is still under active development: methods and behaviors can still evolve until the feature is complete.